core/src/components/avatar/avatar.ionic.scss

```diff
@@ -171,3 +171,9 @@
 :host(.avatar-xxsmall) ::slotted(ion-badge.badge-vertical-bottom:empty) {
   transform: translate(globals.$ion-scale-100, calc(globals.$ion-scale-100));
 }
+
+// Avatar Disabled
+// --------------------------------------------------
+:host(.avatar-disabled) {
+  opacity: 0.6;
```

Comment:
We should be using the design token here: `$ion-state-disabled`

Reply:
I'll use it